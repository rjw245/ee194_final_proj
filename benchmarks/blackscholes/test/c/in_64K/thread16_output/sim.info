{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread16_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes',
             '16',
             '../../../inputs/in_64K.txt',
             'thread16_output/output.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12786.141211,
            132.831806,
            137352,
            0,
            0,
            0,
            225145,
            0,
            0,
            0,
            15456,
            0,
            0,
            0,
            18774278,
            5293064),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/c/in_64K/thread16_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes 16 ../../../inputs/in_64K.txt thread16_output/output.txt'],
 't_elapsed': 8243.59886097908,
 't_start': 1461613856.39606,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 1093206016L}
