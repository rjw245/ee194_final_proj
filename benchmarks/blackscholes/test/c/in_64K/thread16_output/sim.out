                                     | Core 0      | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7    
  Instructions                       |  1338817602 |  593153649 |  593175749 |  593167349 |  593168103 |  593159095 |  593151803 |  593166703
  Cycles                             |   863797760 |  863797760 |  863797760 |  863797760 |  863797760 |  863797760 |  863797760 |  863797760
  IPC                                |        1.55 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69
  Time (ns)                          |   324736000 |  324736000 |  324736000 |  324736000 |  324736000 |  324736000 |  324736000 |  324736000
  Idle time (ns)                     |      232821 |  171624691 |  171719193 |  171781433 |  170883374 |  171765517 |  171219635 |  171443279
  Idle time (%)                      |        0.1% |      52.9% |      52.9% |      52.9% |      52.6% |      52.9% |      52.7% |      52.8%
Branch predictor stats               |             |            |            |            |            |            |            |           
  num correct                        |   153049731 |   34923579 |   34915929 |   34917976 |   34879750 |   34938458 |   34892387 |   34914349
  num incorrect                      |     7836767 |     985127 |     992377 |     990330 |    1028360 |     970422 |    1016123 |     993961
  misprediction rate                 |       4.87% |      2.74% |      2.76% |      2.76% |      2.86% |      2.70% |      2.83% |      2.77%
  mpki                               |        5.85 |       1.66 |       1.67 |       1.67 |       1.73 |       1.64 |       1.71 |       1.68
TLB Summary                          |             |            |            |            |            |            |            |           
  I-TLB                              |             |            |            |            |            |            |            |           
    num accesses                     |   188991179 |   66975040 |   66979445 |   66977635 |   66978372 |   66977355 |   66976569 |   66977772
    num misses                       |         114 |         23 |         23 |         23 |         23 |         23 |         23 |         23
    miss rate                        |       0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |        0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |             |            |            |            |            |            |            |           
    num accesses                     |   445714120 |  190176508 |  190181808 |  190179208 |  190179898 |  190178211 |  190175998 |  190179098
    num misses                       |     1562954 |    4024807 |    2812502 |    1591341 |    3966505 |    4023319 |    2812514 |    3631169
    miss rate                        |       0.35% |      2.12% |      1.48% |      0.84% |      2.09% |      2.12% |      1.48% |      1.91%
    mpki                             |        1.17 |       6.79 |       4.74 |       2.68 |       6.69 |       6.78 |       4.74 |       6.12
  L2 TLB                             |             |            |            |            |            |            |            |           
    num accesses                     |     1563068 |    4024830 |    2812525 |    1591364 |    3966528 |    4023342 |    2812537 |    3631192
    num misses                       |        2406 |        542 |        535 |        584 |       1145 |      10616 |       1261 |       1072
    miss rate                        |       0.15% |      0.01% |      0.02% |      0.04% |      0.03% |      0.26% |      0.04% |      0.03%
    mpki                             |        0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.02 |       0.00 |       0.00
Cache Summary                        |             |            |            |            |            |            |            |           
  Cache L1-I                         |             |            |            |            |            |            |            |           
    num cache accesses               |   188991179 |   66975040 |   66979445 |   66977635 |   66978372 |   66977355 |   66976569 |   66977772
    num cache misses                 |      252268 |       1368 |       5170 |       1415 |       5231 |       1432 |       5338 |       5344
    miss rate                        |       0.13% |      0.00% |      0.01% |      0.00% |      0.01% |      0.00% |      0.01% |      0.01%
    mpki                             |        0.19 |       0.00 |       0.01 |       0.00 |       0.01 |       0.00 |       0.01 |       0.01
  Cache L1-D                         |             |            |            |            |            |            |            |           
    num cache accesses               |   445845509 |  190176517 |  190181817 |  190179217 |  190179906 |  190178221 |  190176006 |  190179106
    num cache misses                 |      755298 |     550376 |     548770 |     560834 |     553684 |     559230 |     561958 |     548271
    miss rate                        |       0.17% |      0.29% |      0.29% |      0.29% |      0.29% |      0.29% |      0.30% |      0.29%
    mpki                             |        0.56 |       0.93 |       0.93 |       0.95 |       0.93 |       0.94 |       0.95 |       0.92
  Cache L2                           |             |            |            |            |            |            |            |           
    num cache accesses               |     1007645 |     551748 |     553941 |     562249 |     558922 |     560671 |     567303 |     553621
    num cache misses                 |      334233 |     220711 |     222869 |     221141 |     223830 |     220259 |     223962 |     223743
    miss rate                        |      33.17% |     40.00% |     40.23% |     39.33% |     40.05% |     39.28% |     39.48% |     40.41%
    mpki                             |        0.25 |       0.37 |       0.38 |       0.37 |       0.38 |       0.37 |       0.38 |       0.38
  Cache L3                           |             |            |            |            |            |            |            |           
    num cache accesses               |      334321 |     220716 |     222870 |     221141 |     223837 |     220268 |     223969 |     223749
    num cache misses                 |       71238 |        711 |        625 |        831 |       3995 |       3839 |       4027 |       4041
    miss rate                        |      21.31% |      0.32% |      0.28% |      0.38% |      1.78% |      1.74% |      1.80% |      1.81%
    mpki                             |        0.05 |       0.00 |       0.00 |       0.00 |       0.01 |       0.01 |       0.01 |       0.01
DRAM summary                         |             |            |            |            |            |            |            |           
  num dram accesses                  |       36388 |          0 |          0 |          0 |      36391 |          0 |          0 |          0
  average dram access latency (ns)   |       61.13 |        inf |        inf |        inf |      61.02 |        inf |        inf |        inf
  average dram queueing delay        |        7.70 |       0.00 |       0.00 |       0.00 |       7.59 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |       0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |             |            |            |            |            |            |            |           
  num loads from dram                |        8582 |        554 |        551 |        545 |         44 |         52 |         37 |         51
  num loads from dram cache          |           0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |        2462 |         99 |         24 |        185 |       3260 |       3186 |       3251 |       3291
